.github/workflows/cd.yaml

```
@@ -0,0 +1,82 @@
+        sudo: true
+
+    - name: Checkout BMF repository (specific branch)
+      run: |
+        sudo apt update
+        sudo apt install -y make git pkg-config libssl-dev cmake binutils-dev libgoogle-glog-dev gcc g++ golang wget libgl1
+        sudo apt install -y python3.9 python3-dev python3-pip libsndfile1 libsndfile1-dev
+        # sudo python3 -m pip install timeout_decorator numpy onnxruntime pytest opencv-python librosa
+        
+        git clone https://github.com/JackLau1222/bmf.git
+        cd bmf
+        git checkout fork_by_oc  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)